ingestion/functions/retrieval/retrieval.py

```diff
@@ -112,9 +123,10 @@ def upload_to_s3(file_name, s3_object_key, source_id, upload_id, api_headers):
 
 
 def invoke_parser(
-    parser_arn, source_id, upload_id, api_headers, s3_object_key,
+    env, parser_arn, source_id, upload_id, api_headers, s3_object_key,
         source_url, date_filter):
     payload = {
+        "env": env,
```

Comment:
perhaps passing the API URL to the parser would make parsing lib simpler, although it's already in common_lib so no big deal... up to you.

Reply:
Yeah, I'd considered this. Should have added a comment in the PR description. This approach adds one "layer" as mentioned, but the plus side is that there's a single place this information is encoded. So if the API URL changes, or something, we can just edit and deploy `common_lib`. Not a huge deal, but nicer than having the information embedded in every event, and needing to somehow update or remake them.

Probably not a huge deal either way, but those were my thoughts.

Reply:
fwiw that information would still be encoded in one place (retrieval.py) if we moved to a url param sent in the event to the parsing layers. But yeah no big deal either way. I just feel like layers in lambda add a layer of complexity (who would have thought?...) especially around weird import tricks that we have to do... anyway, fine :)